src/in/2018-01/03/07.md

```diff
@@ -1,7 +1,18 @@
----
-title:  Pendalaman
-date:   19/01/2018
----
-
-### <center>Kami sedang mengerjakan pelajaran ini</center>
-<center>Silahkan kembali lagi nanti</center>
\ No newline at end of file
+---
+title:  Pendalaman

```

Comment:
@tarigancana I think title is missing here

Reply:
that's on line 2 is the title